routers/api/v1/repo/release.go

```diff
@@ -68,6 +68,47 @@ func GetRelease(ctx *context.APIContext) {
 	ctx.JSON(http.StatusOK, convert.ToRelease(release))
 }
 
+// GetLatestRelease gets the latest stable release of a repository
```

Comment:
```suggestion
// GetLatestRelease gets the most recent non-prerelease, non-draft release of a repository, sorted by created_at
```
This is similar to how GitHub words it, and I think it should make it less ambiguous for users